website/templates/project/register.mako

```diff
@@ -30,7 +30,6 @@
             <div data-bind="foreach: {data: page.questions, as: 'question'}">
               <div class="row">
                 <h4 data-bind="attr: {id: question.id}, text: question.title"></h4>
-                <small><em data-bind="text: question.description"></em></small>
```

Comment:
@barbour-em this may be better to fix in the previewQuestion method-- in particular don't include description the in the preview for questions with only depth 1 in their structure.